src/components/ui/DataGrid/SingleActionDataGrid.svelte

```diff
@@ -184,6 +184,8 @@
         </ContextMenu.Item>
       </div>
     {/if}
+
+    <slot name="context-menu-alt" />
```

Comment:
If the new slot is removed from BulkActionDataGrid, then please remove from here as well.